{'description': {'en': 'Gentoo cross-compilation overlay for embedded devices'},
 'feed': ['https://cgit.gentoo.org/proj/embedded-cross.git/atom/'],
 'homepage': 'https://cgit.gentoo.org/proj/embedded-cross.git/',
 'name': 'embedded-cross',
 'owner': [{'email': 'armin76@gentoo.org',
            'name': u'Ra\xfal Porcel',
            'type': 'project'}],
 'quality': 'experimental',
 'source': [{'type': 'git',
             'uri': 'git://anongit.gentoo.org/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'https://anongit.gentoo.org/git/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'git+ssh://git@git.gentoo.org/proj/embedded-cross.git'}],
 'status': 'official'}
$ pmaint sync embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Already up-to-date.
*** syncing embedded-cross
*** synced embedded-cross
 * Sync succeeded
$ pmaint regen -t 2 embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.061 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.063 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: app-editors/vim-core-7.1.330 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'

!!! ERROR: app-editors/vim-core-7.2.442 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: app-editors/vim-7.3 failed.
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: x11-libs/gtk+-2.14.5 failed.
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.062 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '160' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'

!!! ERROR: app-editors/vim-core-7.0.235 failed.
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '176' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'

!!! ERROR: app-editors/vim-core-7.0.243 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!        vim-core-7.2.442.ebuild, line   6:  Called inherit 'vim'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!        vim-core-7.1.330.ebuild, line   5:  Called inherit 'vim'

!!! ERROR: x11-drivers/xf86-video-glamo-9999 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed


!!! ERROR: app-editors/vim-core-7.1.123 failed.
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed


!!! ERROR: app-editors/vim-core-7.3 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: app-editors/vim-core-7.2.303 failed.

!!! ERROR: x11-drivers/xf86-video-glamo-20100626 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                 vim-7.3.ebuild, line   7:  Called inherit 'vim'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                 ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             gtk+-2.14.5.ebuild, line   7:  Called inherit 'gnome.org' 'flag-o-matic' 'eutils' 'libtool' 'virtualx'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '228' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'virtualx'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '187' 'depend'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                 ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!        vim-core-7.0.235.ebuild, line   5:  Called inherit 'vim'

!!! ERROR: x11-drivers/xf86-input-tslib-0.0.5 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!                 ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '248' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!                virtualx.eclass, line  38:  Called die
!!! The die message:
!!!  virtualx eclass require EAPI=2 or newer.

!!!              ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sci-geosciences/gpsd-2.37 failed.

!!! ERROR: app-misc/mdbus-9999 failed.
!!!        vim-core-7.0.243.ebuild, line   5:  Called inherit 'vim'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!                 ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!              ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!   xf86-video-glamo-9999.ebuild, line  10:  Called inherit 'x-modular'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!                         ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!        vim-core-7.2.303.ebuild, line   6:  Called inherit 'vim'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '178' 'depend'

!!! ERROR: dev-lang/vala-0.8.1.1 failed.
!!!              ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '237' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '165' 'depend'
!!!            vim-core-7.3.ebuild, line   7:  Called inherit 'vim'
!!!                         ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!        vim-core-7.1.123.ebuild, line   5:  Called inherit 'vim'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!              ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 173' for eclass vim

received unknown com: export CATEGORY='dev-lang' PR=r5 PVR='5.1.4-r5' PV='5.1.4' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/lua/lua-5.1.4-r5.ebuild' P='lua-5.1.4' PF='lua-5.1.4-r5' PN=luafailed
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
Exception in thread Thread-9:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sci-geosciences/gpsd/gpsd-2.37.ebuild'
!!!   xf86-video-glamo-20100626.ebuild, line  16:  Called inherit 'x-modular'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: x11-wm/enlightenment-0.16.999.061 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                      ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-input-tslib/xf86-input-tslib-0.0.5.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 179' for eclass vim

received unknown com: export CATEGORY='net-libs' PR=r0 PVR='2.8.5' PV='2.8.5' EBUILD='/home/mgorny/repos/embedded-cross/net-libs/gnutls/gnutls-2.8.5.ebuild' P='gnutls-2.8.5' PF='gnutls-2.8.5' PN=gnutlsfailed
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sci-geosciences/gpsd/gpsd-2.37.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!               x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '179' 'depend'
!!!                  ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!                      ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-input-tslib/xf86-input-tslib-0.0.5.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sci-geosciences/gpsd/gpsd-2.37.ebuild'
!!!                  ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!               gpsd-2.37.ebuild, line   5:  Called inherit 'eutils' 'distutils' 'autotools'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!          isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-input-tslib/xf86-input-tslib-0.0.5.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 

!!! ERROR: dev-util/strace-4.5.15 failed.

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.5 failed.
!!!                  ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.1 failed.
!!!                   x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 193' for eclass distutils

received unknown com: export CATEGORY='net-wireless' PR=r0 PVR=9999 PV=9999 EBUILD='/home/mgorny/repos/embedded-cross/net-wireless/wmiconfig/wmiconfig-9999.ebuild' P='wmiconfig-9999' PF='wmiconfig-9999' PN=wmiconfigpath
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe
!!! Call stack:

!!!   xf86-input-tslib-0.0.5.ebuild, line   7:  Called inherit 'toolchain-funcs' 'eutils' 'linux-info' 'x-modular'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                      ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                  ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '245' 'depend'

!!! ERROR: x11-terms/vala-terminal-1.2 failed.
!!!                      ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'linux-info'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!              mdbus-9999.ebuild, line   5:  Called inherit 'git'
!!!                  ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.330
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.2.442
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.235
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.243
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-9999
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.123
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-core-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: net-libs/gnutls-2.8.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-core-7.2.303
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: dev-lang/lua-5.1.4-r5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-20100626
!!! caught exception Metadata Exception: pkg ebuild src: sci-geosciences/fso-gpsd-0.8, attr eapi
error: parsed eapi doesn't match sourced eapi while processing ebuild src: sci-geosciences/fso-gpsd-0.8
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-drivers/xf86-input-tslib-0.0.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-geosciences/gpsd-2.37
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/gtk+-2.14.5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-misc/mdbus-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/pkgconfig-0.23
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/perl-5.10.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: x11-base/xglamo-20081205-r2
!!! caught exception Metadata Exception: pkg ebuild src: x11-wm/enlightenment-0.16.999.061, attr eapi
error: parsed eapi doesn't match sourced eapi while processing ebuild src: x11-wm/enlightenment-0.16.999.061
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/strace-4.5.18
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/gnutls-2.8.6
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/strace-4.5.15
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-wireless/wmiconfig-9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: !!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                      ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!               ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 179' for eclass linux-info

received unknown com: export CATEGORY='dev-lang' PR=r0 PVR='2.6.4' PV='2.6.4' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.6.4.ebuild' P='python-2.6.4' PF='python-2.6.4' PN=pythonpath
Exception in thread Thread-14:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                  ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                      ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'

!!! ERROR: sys-mobilephone/fsodatad-0.1.0 failed.

!!! ERROR: x11-terms/vala-terminal-1.1.1 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                      ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '285' 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: dev-lang/perl-5.8.8-r10 failed.
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '184' 'depend'
!!!                      ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '285' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                      ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                          ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061.ebuild'
!!!            vala-0.8.1.1.ebuild, line   7:  Called inherit 'gnome2'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: x11-apps/xdm-1.1.8 failed.
!!!                              ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                          ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '208' 'depend'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '218' 'depend'
!!!                      ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-util/strace/strace-4.5.15.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gnome2'
!!!                              ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!                              ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '203' 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-util/strace/strace-4.5.15.ebuild'
!!!              isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '182' 'depend'
!!!                  isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!                              ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                  gnome2.eclass, line  19:  Called die
!!! The die message:
!!!  EAPI=1 is not supported

!!!                  isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-util/strace/strace-4.5.15.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!   enlightenment-0.16.999.061.ebuild, line   8:  Called inherit 'enlightenment'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-9999.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20091021.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.8.8-r10.ebuild'
!!!   freesmartphone-framework-0.5.1.ebuild, line   6:  Called inherit 'git'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'

!!! ERROR: sys-mobilephone/libframeworkd-glib-0.1 failed.
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '164' 'depend'
!!!   freesmartphone-framework-0.5.5.ebuild, line   5:  Called inherit 'distutils'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: sys-mobilephone/freesmartphone-framework-20091021 failed.

!!! ERROR: sys-mobilephone/freesmartphone-framework-9999 failed.
!!!           strace-4.5.15.ebuild, line   5:  Called inherit 'flag-o-matic' 'autotools'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.8.8-r10.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!       vala-terminal-1.2.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment'
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'flag-o-matic'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!                   ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 183' for eclass enlightenment

!!!                       ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='2.0.0' PV='2.0.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild' P='apm2-2.0.0' PF='apm2-2.0.0' PN=apm2path
!!!              ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                         ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                       ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 205' for eclass distutils

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.0.0' PV='1.0.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild' P='fso-alsa-1.0.0' PF='fso-alsa-1.0.0' PN='fso-alsa'path
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe


!!! ERROR: sys-mobilephone/fso-raw-1.0.0 failed.
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.8.8-r10.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 210' for eclass flag-o-matic

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.1' PV='0.9.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild' P='fso-abyss-0.9.1' PF='fso-abyss-0.9.1' PN='fso-abyss'path
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!     vala-terminal-1.1.1.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!              ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                         ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-apps/xdm/xdm-1.1.8.ebuild'

!!! ERROR: sys-mobilephone/fsodeviced-0.9.4 failed.
!!!          perl-5.8.8-r10.ebuild, line   5:  Called inherit 'eutils' 'alternatives' 'flag-o-matic' 'toolchain-funcs' 'multilib'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!          fsodatad-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '245' 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '276' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-apps/xdm/xdm-1.1.8.ebuild'
!!!                         ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '296' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: sys-mobilephone/fsotdld-0.3.0 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'alternatives'
!!!                     ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-apps/xdm/xdm-1.1.8.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 198' for eclass alternatives

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.5.0' PV='0.5.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild' P='fsogsmd-0.5.0' PF='fsogsmd-0.5.0' PN=fsogsmdpath
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '200' 'depend'
Exception in thread Thread-15:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                         ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!                      ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!               xdm-1.1.8.ebuild, line   8:  Called inherit 'multilib' 'x-modular' 'pam'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '213' 'depend'
!!!                             ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-9999.ebuild'
!!!                                 ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20091021.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!                      ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 208' for eclass x-modular

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.4' PV='0.9.4' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild' P='fsousaged-0.9.4' PF='fsousaged-0.9.4' PN=fsousagedpath
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                                 ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20091021.ebuild'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '198' 'depend'
!!!                             ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-9999.ebuild'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!          isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!                     isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20091021.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!                 isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-9999.ebuild'
!!!   libframeworkd-glib-0.1.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!           fso-raw-1.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!   freesmartphone-framework-20091021.ebuild, line   6:  Called inherit 'git'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   freesmartphone-framework-9999.ebuild, line   6:  Called inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!               ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'

!!! ERROR: sys-mobilephone/fsomusicd-0.0.0 failed.

!!! ERROR: sys-mobilephone/libframeworkd-glib-20090227 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!               ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!           ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!                          ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!        fsodeviced-0.9.4.ebuild, line   6:  Called inherit 'cornucopia_git'

!!! ERROR: sys-mobilephone/libframeworkd-glib-99999999 failed.
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 306' for eclass base

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090614 PV=20090614 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090614.ebuild' P='libframeworkd-phonegui-efl-20090614' PF='libframeworkd-phonegui-efl-20090614' PN='libframeworkd-phonegui-efl'path
!!!             ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
Exception in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! Call stack:
!!!                      ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                   ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
dev-lang/perl-5.8.8-r10
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/python-2.6.4
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsodatad-0.1.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fsoinitd-0.1.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.1.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-apps/xdm-1.1.8
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/vala-0.8.1.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libframeworkd-glib-0.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-20091021
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/apm2-2.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fso-raw-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fso-alsa-1.0.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090724
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fso-abyss-0.9.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodeviced-0.9.4
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsotdld-0.3.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsogsmd-0.5.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfsobasics-0.9.9
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsousaged-0.9.4
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfsoresource-0.1.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsomusicd-0.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090227
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libgsm0710mux-0.9.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090614
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/l
!!! ERROR: sys-mobilephone/libframeworkd-glib-20090614 failed.
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                   ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-99999999.ebuild' P='libframeworkd-phonegui-99999999' PF='libframeworkd-phonegui-99999999' PN='libframeworkd-phonegui'path
Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!           fsotdld-0.3.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                   ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '266' 'depend'
!!!                   ebuild-daemon.bash, line 457:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '208' 'depend'
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 255' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='2010.05.11.2' PV='2010.05.11.2' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfso-glib/libfso-glib-2010.05.11.2.ebuild' P='libfso-glib-2010.05.11.2' PF='libfso-glib-2010.05.11.2' PN='libfso-glib'path
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                   ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                   ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '266' 'depend'
!!!             ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                   ebuild-daemon.bash, line 440:  Called __ebd_process_metadata '266' 'depend'
!!!                           ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!                   ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                   ebuild-daemon.bash, line 373:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!                           ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!                           ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!                           ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!                           ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!                           ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!               isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!         fsomusicd-0.0.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!               isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!               isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!   libframeworkd-glib-20090614.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!   libframeworkd-glib-20090227.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 213' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.2.1' PV='1.2.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libgsm0710/libgsm0710-1.2.1.ebuild' P='libgsm0710-1.2.1' PF='libgsm0710-1.2.1' PN=libgsm0710path
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   libframeworkd-glib-99999999.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 228' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.2' PV='0.9.2' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libgsm0710mux/libgsm0710mux-0.9.2.ebuild' P='libgsm0710mux-0.9.2' PF='libgsm0710mux-0.9.2' PN=libgsm0710muxfailed
Exception in thread Thread-16:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 231' for eclass base

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-ui/libphone-ui-99999999.ebuild' P='libphone-ui-99999999' PF='libphone-ui-99999999' PN='libphone-ui'path
Exception in thread Thread-11:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 246' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-utils/libphone-utils-99999999.ebuild' P='libphone-utils-99999999' PF='libphone-utils-99999999' PN='libphone-utils'failed
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 494, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 836, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 769, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 459, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

ibphone-utils-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/mioctl-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfso-glib-2010.05.11.2
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/mterm2-1.9.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libgsm0710-1.2.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libphone-ui-shr-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libphone-ui-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phonefsod-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/mdbus2-2.0.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phoneuid-99999999
Error running an operation:
 Unhandled exception in regen_cache
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d646788> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d646680> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d646578> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d646470> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d646368> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d646260> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d646158> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d646050> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d69df18> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d69de10> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d69dd08> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d69dc00> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d69daf8> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d69d9f0> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d69d8e8> ignored
Exception IOError: IOError(32, 'Broken pipe') in <functools.partial object at 0x7faa1d69d260> ignored
 * Cache regen failed with 246
